modules/stanford_private_page_access/stanford_private_page_access.strongarm.inc

```diff
@@ -14,20 +14,7 @@ function stanford_private_page_access_strongarm() {
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
   $strongarm->name = 'content_access_stanford_private_page';
-  $strongarm->value = array(
-    'view_own' => array(
-      0 => 10,
-      1 => 9,
-      2 => 7,
-      3 => 8,
-    ),
-    'view' => array(
-      0 => 10,
-      1 => 9,
-      2 => 7,
-      3 => 8,
-    ),
-  );
+  $strongarm->value = variable_get('content_access_stanford_private_page', array());
```

Comment:
If you are going to set it to whatever it is already set to, why track this in the feature at all?